internal/api/schemas/v1.yaml

```diff
@@ -271,6 +439,178 @@ components:
         privateKeyField:
           type: string
           minLength: 1
+    HTTPRoutePage:
+      type: object
+      required:
+      - routes
+      properties:
+        routes:
+          type: array
+          items:
+            "$ref": "#/components/schemas/HTTPRoute"
+        meta:
+          type: object
+    GatewayReference:
+      type: object
+      required:
+      - name
+      - namespace
+      properties:
+        name:
+          type: string
+          minLength: 1
+        namespace:
+          type: string
+          minLength: 1
+    HTTPRoute:
+      type: object
+      required:
+      - name
+      - namespace
+      - gateways
+      properties:
+        name:
+          type: string
+          minLength: 1
+        namespace:
+          type: string
+          minLength: 1
+        gateways:
+          type: array
+          items:
+            "$ref": "#/components/schemas/GatewayReference"
+          minItems: 1
+        hostnames:
+          type: array
+          items:
+            type: string
+        rules:
+          type: array
+          items:
+            "$ref": "#/components/schemas/HTTPRouteRule"
+    HTTPRouteRule:
+      type: object
+      properties:
+        matches:
+          type: array
+          items:
+            "$ref": "#/components/schemas/HTTPMatch"
+        filters:
+          "$ref": "#/components/schemas/HTTPFilters"
+        services:
+          type: array
+          items:
+            "$ref": "#/components/schemas/HTTPService"
+    HTTPService:
+      type: object
+      required:
+      - name
+      properties:
+        name:
+          type: string
+        namespace:
+          type: string
+        weight:
+          type: number
+        filters:
+          "$ref": "#/components/schemas/HTTPFilters"
+    HTTPFilters:
```

Comment:
Worth going ahead and adding path prefix replace filter since we support it in K8s land now?

Reply:
Good call, I'll do that in a follow-up.